### DISCUSSION TITLE
Muting Firefox when FL Studio begins playing sound not working on my end

### DISCUSSION THREAD

**Discussion by Grenbur** [File-New-Project/EarTrumpet#827] [Q&A]
I want the Firefox browser to automatically mute when FL Studio plays audio, and unmute when that audio stops. It's not working and I'm not sure why. 

I'm also unable to attach a copy of my actions as a file. Could someone help?

**Comment by riverar** [comment]
I created issue https://github.com/File-New-Project/EarTrumpet/issues/828 for you. You can attach your action there!
